docs/install-docker.md

```diff
@@ -241,3 +241,145 @@ make clean-conda
 make clean-bin
 ```
 
+# Mac M1 Installation
+
+### 1) Step1: Install OS Dependencies
+
+Mac users can get [docker desktop](https://docs.docker.com/desktop/mac/apple-silicon/). Also installed rosetta and used Docker Compose V2 as suggested at the moment.
+
+- **Optional**: these installations are not mentioned but might be needed:
+  - Install [brew](https://brew.sh/)
+  - Install md5sha1sum (`brew install md5sha1sum`)
+  - Install PostgreSQL (`brew install postgresql`)
+
+### Step 2: Initialize CanDIGv2 Repo
+
+```bash
+# 1. initialize repo and submodules
+git clone -b develop https://github.com/CanDIG/CanDIGv2.git
+cd CanDIGv2
+git submodule update --init --recursive
+
+# 2. copy and edit .env with your site's local configuration
+cp -i etc/env/example.env .env
+```
+
+- Edit `VENV_OS=arm64mac` for the .env file:
+
+```bash
+# options are [linux, darwin, arm64mac]
+VENV_OS=arm64mac
+VENV_NAME=candig
+```
+
+- Continue to run `make`
+
+```bash
+# 3. fetch binaries and initialize candig virtualenv
+make bin-all
+make init-conda
+```
+
+- To activate conda env, do the following:
+
+```bash
+conda env list
+# Copy the whole path that contains `/envs/candig`
+conda activate {path_to_folder}/CanDIGv2/bin/miniconda3/envs/candig
+```
+
+- Note: The reason we cannot activate it automatically on Mac was described in this [post](https://stackoverflow.com/questions/57527131/conda-environment-has-no-name-visible-in-conda-env-list-how-do-i-activate-it-a). If `conda env` is not in the root folder, it won't have a name.
+
+### Step 3: Initialize CanDIGv2 (Docker)
+
+- Make sure you are in `candig` virtual environment (activate it in previous step)
+
+```bash
+make init-docker
+```
+
+### Step 4: Deploy CanDIGv2 Services (Compose)
+
+```bash
+make compose
+```
+
+### Step 5: Update hosts
+
+- Get the local IP address in the terminal:
+
+```bash
+dig -4 TXT +short o-o.myaddr.l.google.com @ns1.google.com
+```
+
+- Then edit your /etc/hosts file:
+
+```bash
+sudo nano /etc/hosts
+```
+
+- Add the IP address to the end of the file so it look like this:
+
+```bash
+# Other settings
+xx.xx.xx.xx  docker.localhost
```

Comment:
I thought that you used host.docker.internal?

Reply:
yea, I have a note below for local development. If it is confusing I can fix it

Reply:
It just seems to me that if you're using an M1 Mac, you're doing local development.

Reply:
but Brennan didn't run into this problem. If he didn't have to use host.docker.internal then I will leave the option there

Reply:
@brouillette, you were able to use docker.localhost on your M1 setup? Do you know if you had to change any settings anywhere to make that work? @kcranston, what did you use?

Reply:
Yes

``` 
# .env

...

CANDIG_DOMAIN=docker.localhost
CANDIG_AUTH_DOMAIN=auth.docker.localhost
...
```

Also, adjusting the `/etc/hosts` to point to the machine's local LAN IP helped .. 
Vault wasn't liking trying to resolve `0.0.0.0` or `localhost` from `docker.localhost` or `auth.docker.localhost` 
```
# /etc/hosts

...
# CanDIGv2

192.168.1.XX docker.localhost
192.168.1.XX auth.docker.localhost

...
```

Reply:
So it's still a mystery why Son's machine didn't default to using docker.localhost but used host.docker.internal instead, huh?

Reply:
this is fixed in #170